src/build_system.ml

```diff
@@ -250,59 +255,78 @@ module Alias0 = struct
   let fully_qualified_name t = Path.relative t.dir t.name
 
   let stamp_file t =
-    Path.relative (Path.insert_after_build_dir_exn t.dir ".aliases") (t.name ^ suffix)
+    Path.relative (Path.insert_after_build_dir_exn t.dir ".aliases")
+      (t.name ^ suffix)
 
   let dep t = Build.path (stamp_file t)
 
+  let find_dir_specified_on_command_line ~dir ~file_tree =
+    match File_tree.find_dir file_tree dir with
+    | None ->
+      die "From the command line:\n\
```

Comment:
Same remark about `die` - can we inject a loc here?

Reply:
This is from the command line as well, so we don't have a more precise location either